src/agent/rustjail/Cargo.toml

```diff
@@ -31,6 +31,10 @@ tokio = { version = "1.2.0", features = ["sync", "io-util", "process", "time", "
 futures = "0.3"
 async-trait = "0.1.31"
 inotify = "0.9.2"
+libseccomp = "0.1.2"
```

Comment:
Let's include this only when `seccomp` feature enabled.
See examples here: https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

Reply:
It is cool.
Thanks for telling me.